packages/readable-js-assets-webpack-plugin/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "@wordpress/readable-js-assets-webpack-plugin",
+	"version": "1.0.0",
+	"description": "Generate a readable JS file for each JS asset.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"gutenberg",
+		"webpack",
+		"readable",
+    "minimizer"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/HEAD/packages/readable-js-assets-webpack-plugin/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/readable-js-assets-webpack-plugin"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"engines": {
+		"node": ">=12.0"
+	},
+	"files": [
+    "index.js"
+	],
+	"main": "index.js",
+	"peerDependencies": {
+		"webpack": "^4.8.3 || ^5.0.0"
```

Comment:
We'll need to update the plugin for webpack 5. Instead of the `additionalAssets` hook, we'll likely be hooking into something else. The `terser-webpack-plugin` also uses different hooks for v4 and v5.

Reply:
Hmm, not sure I understood. Should we do this now or only when we migrate to v5? AFAIK we're still using v4.x, right?